app-modules/portal/src/Http/Middleware/AuthenticateIfRequiredByPortalDefinition.php

```diff
@@ -51,8 +54,14 @@ public function handle(Request $request, Closure $next): Response
             return $next($request);
         }
 
-        if (! auth('sanctum')->check()) {
-            abort(403);
+        $user = app(Guard::class)($request);
```

Comment:
Would `auth('sanctum')->user()` work here? What is the purpose of directly instantiating the guard class?